### ISSUE TITLE
Add species statistics and estimates to the classification tree

### ISSUE THREAD

**Issue by gdower** [CatalogueOfLife/portal#12]
We need to add species statistics to the classification tree, similar to the legacy interface:

![image](https://user-images.githubusercontent.com/8573609/77562933-a1f7bb80-6e8e-11ea-8277-466924a0ff5d.png)

**Comment by mdoering** [comment]
Use `est.sp.` as the estimation label, but also show number of living and extinct species in the CoL.
Partially blocked by https://github.com/CatalogueOfLife/backend/issues/834

**Comment by mdoering** [comment]
@olafbanki @thomasstjerne @gdower @yroskov are the actual species counts, not estimates, important for the first release? It will require [backend work](#834) so we need to schedule that

**Comment by yroskov** [comment]
How final release will look is important for me, not the first one.

**Comment by mdoering** [comment]
duplicate of #23
